modules/exploits/windows/http/ws_ftp_rce_cve_2023_40044.rb

```diff
@@ -0,0 +1,171 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Powershell
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Progress Software WS_FTP Unauthenticated Remote Code Execution',
+        'Description' => %q{
+          This module exploits an unsafe .NET deserialization vulnerability to achieve unauthenticated remote code
+          execution against a vulnerable WS_FTP server running the Ad Hoc Transfer module. All versions of WS_FTP server
+          prior to 8.7.4 and 8.8.2 are vulnerable to this issue. The vulnerability was originally discovered by AssetNote.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'sfewer-r7', # MSF Exploit & Rapid7 Analysis
+        ],
+        'References' => [
+          ['CVE', '2023-40044'],
+          ['URL', 'https://attackerkb.com/topics/bn32f9sNax/cve-2023-40044/rapid7-analysis'],
```

Comment:
```suggestion
          ['URL', 'https://attackerkb.com/topics/bn32f9sNax/cve-2023-40044/rapid7-analysis'],
          ['URL', 'https://www.assetnote.io/resources/research/rce-in-progress-ws-ftp-ad-hoc-via-iis-http-modules-cve-2023-40044'],
```

Reply:
Added the AssetNote reference in [2eacb75](https://github.com/rapid7/metasploit-framework/pull/18414/commits/2eacb75feb72a87bf68bb48262f0c436208e15c5)